charts/kube-state-metrics/OWNERS

```diff
@@ -1,6 +1,8 @@
 approvers:
 - tariq1890
 - mrueg
+- dotdc
```

Comment:
Can you remove this file instead? It got accidentally copied over when we transferred the chart here.

Reply:
Sure, done in https://github.com/prometheus-community/helm-charts/pull/1675/commits/85c66e54dbfc9a26f89d20aa7a15e214b6b969e6